docs/diagrams/AddPersonSequenceDiagram.puml

```diff
@@ -27,16 +27,17 @@ create AddPersonCommand
 AddPersonCommandParser -> AddPersonCommand : parse(cmd)
 activate AddPersonCommand
 
-AddPersonCommand --> AddPersonCommandParser
+AddPersonCommand --> AddPersonCommandParser : a
 deactivate AddPersonCommand
 
-AddPersonCommandParser --> AddressBookParser
+AddPersonCommandParser --> AddressBookParser : a
 deactivate AddPersonCommandParser
 
-AddressBookParser --> LogicManager : c
+AddressBookParser --> LogicManager : a
 deactivate AddressBookParser
+destroy AddPersonCommandParser
 
-LogicManager -> AddPersonCommand : execute()
+LogicManager -> AddPersonCommand : execute(b)
 activate AddPersonCommand
 
 AddPersonCommand -> ModelManager : addPerson(Person)
```

Comment:
I think it's better to add the `UnqiuePersonList` in this sequence diagram to be consistent with add event sequence diagram

Reply:
ah time is a little tight so i think will have to leave as is for now